.github/workflows/libmacchina.yml

```diff
@@ -1,4 +1,4 @@
-name: libmacchina test
+name: CI
 
 on:
```

Comment:
Is there a reason why the workflows are only running on the "main" branch?

Reply:
Not really, I'll have it run for all of them.

Reply:
Awesome :)